tests/crm/test_crm.py

```diff
@@ -808,10 +808,21 @@ def test_crm_nexthop_group(duthosts, enum_rand_one_per_hwsku_frontend_hostname,
     verify_thresholds(duthost, asichost, crm_cli_res=redis_threshold, crm_cmd=get_nexthop_group_stats)
 
 
-def test_acl_entry(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_frontend_asic_index, collector):
+def test_acl_entry(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_frontend_asic_index, collector, tbinfo):
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
     asichost = duthost.asic_instance(enum_frontend_asic_index)
     asic_collector = collector[asichost.asic_index]
+
+    if duthost.facts["asic_type"] == "marvell":
+        # Remove DATA ACL Table and add it again with ports in same port group
+        ACL_TABLE_NAME = "DATAACL"
+        mg_facts = duthost.get_extended_minigraph_facts(tbinfo)
+        ports = ",".join(mg_facts["minigraph_portchannels"].keys())
```

Comment:
What are original ports and what will be new ports that binding with DATAACL table? Could you help add some example data in your PR description or comments? Then we can understand it easily 

And can you help add more details? such as what is the limitation of marvell asic? And what is the definition of "same port group?"?

Reply:
Hi @Blueve 

    This specific change is to align Marvell SAI design.
When an acl is bind to set of ports, there is a possibility that more than one rule is configured in ASIC for single user-configured rule.
This is because of mapping the ports to corresponding port bit-map in Marvell SAI.
For same user configrued acl rule, if port-number falls different port-bitmap, SAI will replicate the rule.
We ensure that the ports used in this TC will fall under same port-bitmap to have one-to-one mapping of CRM Used-count and available-count.

In this test case the  DATAACL table is bound to ports 46, 47, 48, 49, 50 and 51
that's why deleted the ACL TABLE first and then re-added it with ports 48,49,50 and 51 so that table will fall under same port-bit map.
and at last recovered the original config of TC 
In future we will introduce new TC to community to cover ports falling under different port-bitmaps.